Manual/manJHUGenerator.tex

```diff
@@ -1366,13 +1366,20 @@ \section{JHU Generator Coupling Converter (JHUGenLexicon)}
 
 \section{Release notes}
 \noindent 
-In going from \verb|v7.5.1| to \verb|v7.5.2| (October 1 2022), the updates are as follows:
+In going from \verb|v7.5.2| to \verb|v7.5.3| (January 25, 2023), the updates are as follows:
+\noindent
+\begin{itemize}
+\item Added width scheme 4 (running) in JHUGen and MCFM-JHUGen processes
+\item Add Zff couplings in MCFM-JHUGen ggH offshell process
```

Comment:
I would leave this second item to a different PR stage. Right now, this PR only implements a different propagator scheme. I would also clarify this new scheme a bit further and cite the reference in the manual. Please be specific about section and equation numbers if the reference document is long.